<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="xlab_layout">
     <item>
      <widget class="QCheckBox" name="xlabCheckBox">
       <property name="text">
        <string>set xlabel?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>xlabel:</string>
       </property>
       <property name="buddy">
        <cstring>xlab_le</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xlab_le">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ylab_layout">
     <item>
      <widget class="QCheckBox" name="ylabCheckBox">
       <property name="text">
        <string>set ylabel?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ylabel:</string>
       </property>
       <property name="buddy">
        <cstring>ylab_le</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ylab_le">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="xlim_layout">
     <item>
      <widget class="QCheckBox" name="xlimCheckBox">
       <property name="toolTip">
        <string>x axis limits. If not checked, the function tries to set the xaxis limits to some sensible values.</string>
       </property>
       <property name="text">
        <string>set xlims?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>[</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xlimLowSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="xlimHighSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ylim_layout_2">
     <item>
      <widget class="QCheckBox" name="ylimCheckBox">
       <property name="toolTip">
        <string>y axis limits. If not checked, the function tries to set the yaxis limits to some sensible values.</string>
       </property>
       <property name="text">
        <string>set ylims?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ylim_layout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>[</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ylimLowSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ylimHighSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="binwidth_layout">
     <item>
      <widget class="QCheckBox" name="binwidth_checkBox">
       <property name="text">
        <string>set bin width?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="binwidth_spinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>(defaults to range/30)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Histogram Bar Colors</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="no_gradient_radiobtn">
          <property name="text">
           <string>No Gradient</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gradient_radiobtn">
          <property name="text">
           <string>gradient</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="fixed_color_layout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Fixed Color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="change_fixed_color_btn">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>change
fill color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>change
outline color</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="gradient_layout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Gradient:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="low_color_change_btn">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 0, 0);color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>change
low color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="high_color_change_btn">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 255);color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>change
high color</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Count legend title:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="count_le">
              <property name="text">
               <string>Count</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
